Automattic/pocket-casts-ios
# [Deselect Chapters] Design

| 📘 Part of: #1396 | 🎨 [Designs](https://www.figma.com/file/hHFpI1RtnW4TRb448Q4Don/Player?type=design&node-id=4383%3A26935&mode=design&t=3kwjoLnHImygBNhh-1) |
|:---:|:---:|

Implements the final design of deselecting chapters and adds a few tweaks.

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/e19d20f7-780f-4e01-bd63-4f5583c61897" width="300">

These are podcasts with chapters you can use to test:

* Clublife: https://pca.st/rLzZCv
* ATP: https://pca.st/accidentaltech

## To test

1. Go to Profile > Settings > Beta Features > `deselectChapters` and turn it on
2. Go to Profile > Settings > Developer > Set to Paid Patron
3. Play any episode with chapters
4. Open the player and go to Chapters
5. Toggle the chapter selection
6. ✅ Compare what you see with the designs in Figma
7. Deselect the chapter that is currently being played
8. Tap "Done"
9. ✅ The current chapter should skip
10. Toggle the chapter selection
11. Deselect all chapters
12. ✅ On the latest selected chapter you should see a message when you try to deselect
13. Select again a few chapters
14. Tap "Done"
15. Tap any chapter
16. ✅ The correct chapter should be played

## Checklist
